adr/0010-web-dashboard-directory-structure-and-contexts.md

```diff
@@ -0,0 +1,80 @@
+# 10. Web dashboard directory structure and contexts
+
+Date: 2023-08-24
+
+## Status
+
+Proposal
+
+## Context
+
+The web dashboard repository, is a `phoenix` application, right now the directory structure and the overall `phoenix` framework convention regarding modules and code organization are not respected.
+With the increasing complexity of the project and the codebase, we need to switch to the `phoenix` framework standards, to be compatible with code generators and framework conventions in general.
+
+The current directory structure is too complex in a lot of aspects, given the fact that it's really difficult to follow where the moving parts are and deciding where to put what is becoming more difficult with the increasing codebase complexity.
+
+The web project is also a `commanded` application, `commanded` has examples and guidelines on how to structure the projects, the directories and the modules, this documentation with the official  `phoenix` guide will be the base principles for this refactoring.
```

Comment:
`The web repository is also a commended application. commanded has examples...`. Better a dot after the 1st phrase.

`...projects, the directories and the modules. This documentation`. Better dot here as well